GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -359,10 +394,7 @@ extension CameraViewController {
             do {
                 try qrDocument.validate()
                 self.showPopup(forQRDetected: qrDocument)
-            } catch let error as DocumentValidationError {
```

Comment:
Why did you remove this?

Reply:
Because it was only printing, I don't find it very useful